### ISSUE TITLE
document.cookies is empty string

### ISSUE THREAD

**Issue by 0xsven** [thereactivestack/react-cookie#49]
I am expecting cookie.save to save a cookie. The following code shows how document.cookies is empty after using the save function. 

```
import cookie from 'react-cookie'

cookie.save('session', 'SESSIONTOKEN')
console.log(document.cookies) // => ""
console.log(cookie.load('session', true)) // => "SESSIONTOKEN"
```

What am I doing wrong or understanding wrong?

**Comment by 0xsven** [comment]
This line:

https://github.com/thereactivestack/react-cookie/blob/master/src/cookie.js#L4

is what breaks it. It assumes to be in node when it's really in the browser.

**Comment by robinkossi** [comment]
I've been having the same problem, it seems like this is a bug introduced in 1.0.0.

Looks like it is indeed the IS_NODE constant not working properly.

I downgraded to v0.4.9 and have no issues, so I think I'll stay on that version for now.

**Comment by eXon** [comment]
Thanks for reporting. This is a duplicate to https://github.com/thereactivestack/react-cookie/issues/48

**Comment by 0xsven** [comment]
document.cookies is still empty string with version 1.0.2 -.-

**Comment by eXon** [comment]
@0xsven should be fixed in 1.0.3 sorry about that

**Comment by 0xsven** [comment]
this library is pretty neat btw.!
Thank's for developing it :)
